File: clients/typescript/src/SystemdManagerStopUnitRequest.ts

export class SystemdManagerStopUnitRequest {
  private _unitName: string;

  constructor(input: {
    unitName: string,
  }) {
    this._unitName = input.unitName;
  }

  get unitName(): string { return this._unitName; }
  set unitName(unitName: string) { this._unitName = unitName; }
}

File: clients/rust/src/systemd_manager_stop_unit_request.rs
// SystemdManagerStopUnitRequest represents a SystemdManagerStopUnitRequest model.
#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub struct SystemdManagerStopUnitRequest {
    #[serde(rename="unit_name")]
    pub unit_name: String,
}

impl SystemdManagerStopUnitRequest {
    pub fn new(unit_name: String) -> SystemdManagerStopUnitRequest {
        SystemdManagerStopUnitRequest {
            unit_name,
        }
    }
}
